src/main/java/seedu/toluist/model/Task.java

```diff
@@ -38,9 +38,14 @@ public Task(String description, LocalDateTime endDateTime) {
     }
 
     public Task(String description, LocalDateTime startDateTime, LocalDateTime endDateTime) {
+        this(description, startDateTime, endDateTime, new TreeSet<Tag>());
+    }
+
+    public Task(String description, LocalDateTime startDateTime, LocalDateTime endDateTime, Collection<Tag> tags) {
```

Comment:
I prefer not to have this constructor here an just use `replaceTags`, because there is a default for tag (empty set). The other 3 constructors exist because semantically it represent floating task, task w/ deadline & event